### ISSUE TITLE
smbpasswd reports failure in occidentalis postinst

### ISSUE THREAD

**Issue by brennen** [adafruit/Adafruit-Pi-Finder#22]
```
Restarting Avahi mDNS/DNS-SD Daemon: avahi-daemon.
Restarting Netatalk Daemons (this will take a while)Stopping Netatalk Daemons: afpd cnid_metad papd timelord atalkd.
..Starting Netatalk services (this will take a while):  cnid_metad afpd.
done.
Mismatch - password unchanged.
Unable to get new password.
Stopping Samba daemons: nmbd smbd.
Starting Samba daemons: nmbd smbd.
```

**Comment by toddtreece** [comment]
@brennen have you changed the default pass for the pi user?

**Comment by brennen** [comment]
Nope - I've been keeping this install as stock as possible for testing bootstrap stuff.

That does maybe raise the question of whether we should mandate or strongly encourage the password change, but yeah, I guess this is really only a bug insofar as someone might notice it and be a little confused.  Probably not worth worrying about.

**Comment by toddtreece** [comment]
I think the bug is here: https://github.com/adafruit/Adafruit-Pi-Finder/blob/master/packages/occidentalis/DEBIAN/postinst#L23

I'll test & fix

**Comment by jakymiwm** [comment]
I just installed this on a fresh PI and I am not able to access the smb share. The credentials are bad.
FYI, my PC is on a domain. I made sure the username was raspberrypi\pi

I think it is in: https://github.com/adafruit/Adafruit-Occidentalis/blob/master/packages/occidentalis/DEBIAN/postinst#L32

**Comment by jakymiwm** [comment]
I did a pull request on Occidentalis repo, not sure if anyone has seen it.
https://github.com/adafruit/Adafruit-Occidentalis/pull/4
Removing the '-ne' and '\n' seem to fix the "Mismatch - password unchanged." error.

**Comment by toddtreece** [comment]
fixed by adafruit/Occidentalis#4
